gitlab/ci/misc.yml

```diff
@@ -79,4 +79,4 @@ detect dump:
         -name "ace" -type d -prune -o
         -type f -print0 | xargs -0 grep --color "dump(" | grep -v "Yaml::dump(")
     - echo $OUT
-    - exit $(($(wc -l <<< $OUT)-1))
+    - exit $(($(wc -l <<< $OUT)-2))
```

Comment:
Note: I added this. If we remove the bench.py file from this repo, we need to revert this to 1 again.